tuned/plugins/plugin_bootloader.py

```diff
@@ -13,10 +13,163 @@
+	file and grub environment files.
+	+
+	The kernel options can be specified by the following syntax:
+	+
+	[subs="+quotes,+macros"]
+	----
+	+cmdline+[replaceable]__suffix__=[replaceable]__arg1__ [replaceable]__arg2__ ... [replaceable]__argN__
+	----
+	+
+	Or with an alternative, but equivalent syntax:
+	+
+	[subs="+quotes,+macros"]
+	----
+	+cmdline+[replaceable]__suffix__=+[replaceable]__arg1__ [replaceable]__arg2__ ... [replaceable]__argN__
+	----
+	+
+	Where [replaceable]__suffix__ can be arbitrary (even empty) alphanumeric
+	string which should be unique across all loaded profiles. Good
+	practice is to use the profile name as the [replaceable]__suffix__
+	(e.g. [option]`cmdline_my_profile`). If there are multiple
+	[option]`cmdline` options with the same suffix (e.g. in the parent
+	profiles), then during the profile load/merge the value which was
+	assigned the last time wins (this is the same as with any other
+	plug-in options). Finaly, the kernel command line is constructed by
+	concatenation of the all resulting [option]`cmdline` options.
+	+
+	It is also possible to remove kernel options by the following syntax:
+	+
+	[subs="+quotes,+macros"]
+	----
+	+cmdline+[replaceable]__suffix__=-[replaceable]__arg1__ [replaceable]__arg2__ ... [replaceable]__argN__
+	----
+	+
+	Such kernel options will be removed (instead of concatenated) during
+	the final kernel command line construction.
+	+
+	.Modifying the kernel command line
+	====
+	For example, to add the [option]`quiet` kernel option to a *TuneD*
+	profile, include the following lines in the [filename]`tuned.conf`
+	file:
+	
+	----
+	[bootloader]
+	cmdline_my_profile=+quiet
+	----
+	
+	An example of a custom profile `my_profile` that adds the
+	[option]`isolcpus=2` option to the kernel command line:
+	
+	----
+	[bootloader]
+	cmdline_my_profile=isolcpus=2
+	----
+	
+	An example of a custom profile `my_profile` that removes the
+	[option]`rhgb quiet` options from the kernel command line (if
+	previously added by *TuneD*):
+	
+	----
+	[bootloader]
+	cmdline_my_profile=-rhgb quiet
+	----
+	====
+	+
+	.Modifying the kernel command line, example with inheritance
+	====
+	For example, to add the [option]`rhgb quiet` kernel options to a
+	*TuneD* profile `profile_1`:
+	
+	----
+	[bootloader]
+	cmdline_profile_1=+rhgb quiet
+	----
+	
+	In the child profile `profile_2` drop the [option]`quiet` option
+	from the kernel command line:
+	
+	----
+	[main]
+	include=profile_1
+	
+	[bootloader]
+	cmdline_profile_2=-quiet
+	----
+
+	The final kernel command line will be [option]`rhgb`. In case the same
+	[option]`cmdline` suffix as in the `profile_1` is used:
+
+	----
+	[main]
+	include=profile_1
+
+	[bootloader]
+	cmdline_profile_1=-quiet
+	----
+
+	It will result in the empty kernel command line because the merge
+	happens at first and the [option]`cmdline_profile_1` gets redefined
+	to just [option]`-quiet` and there is nothing to remove in the final
+	kernel command line processing.
+	====
```

Comment:
It will result in the empty kernel command line because the merge executes and the [option]`cmdline_profile_1` gets redefined to just [option]`-quiet`. Thus there is nothing to remove in the final kernel command line processing.